Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml

```
@@ -144,12 +144,59 @@ script:
       description: ID of the entry containing the file to upload
       isArray: true
       name: upload
-      required: true
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Use XSOAR built-in polling to retrieve the result when it's ready.
+      isArray: false
+      name: polling
+      predefined:
+      - 'true'
+      - 'false'
+    - default: false
+      description: Interval in seconds between each poll.
+      isArray: false
+      name: interval_in_seconds
+      required: false
+      secret: false
+      defaultValue: '60'
+    - default: false
+      description: Used for the inner polling flow, for uploading a file use the 'upload'
+        argument instead
+      isArray: true
+      name: md5
+      required: false
+      secret: false
```

I don't like adding an argument that's only for internal use. If it's really not useful for users at least use `deprecated: true` to hide it (make sure it works).
```suggestion
      description: Used for the inner polling flow, for uploading a file use the 'upload'
        argument instead.
      isArray: true
      name: md5
      required: false
      deprecated: true
      secret: false
```

---

Agreed, wasn't sure whats the best way about that. Added `deprecated: true` (+tested).